service/quote-server/README.md

```diff
@@ -0,0 +1,88 @@
+# Service: CCNP Quote Service
+
+This service will provide quote generated by underlying TEE platform for remote attestation service to verify the integrity and confidentiality of the trusted computing environment and required software environment.
+
+## Introduction
+
+This service provides functionality to fetch quote of underlying TEE platform with nonce as mandatory input and a base64 encoded user data as optional input. The nonce and optional user data will be digested and added into quote for remote attestation to verify the freshness of the quote and the user specified data.
+
+The quote server use Unix domain socket based gRPC to serve the client SDK and the proto buffer message is as bellow:
```

Comment:
1. It's wired to use "Unix". Change to other word?  Linux? Network? ...
2. Separate 2nd sentence.
The proto buffer message definition is as below:

Reply:
UDS is a special term: https://en.wikipedia.org/wiki/Unix_domain_socket